### DISCUSSION TITLE
Dialog maxWidth

### DISCUSSION THREAD

**Discussion by jsmolina** [Telefonica/mistica-design#1955] [Proposals]
### Goal

The idea is to offer providing wider dialogs with some property like mui maxWidth, so we can use screen more efficently.

### Context

Current dialogs provide a non configurable width, which limits the capability of showing some forms or data on desktop.

### Impact

Adding maxwidth property will improve UX for desktop.

### Discussion Questions

- should we use maxwidth or predefined widths? (like t-shirt sizes, xs, sm, md, xl, xxl)

### Do you have a visual proposal?

[Current issue](https://mistica-web.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8AlGBnADhAduglgG4wAyAhgJ4QCuALgHwA6OABG%2B0gEJ227lV1m7YS0wAnfAFsyYit1q8cAXmBCRwrj1wAFCdNlr1wmfjIBaABZksETNUxLGIKKYA2EAOZPDRtjrEY6CoAFACULEpMrL4iLmTuHsGq0TEitPi0rjDwLE4AcjAA7izEYgS4TgA0PqlsMAAetGJkOcE1tWxcEPUwUFEdvl31omRQLjgeKgCMAAwAvv0Dgy6E9ABqMGX4uCwzSAD0K4tL6kgA4hJQLB5kDsBTAExzLJCu1JJ4Kk-HJ6cX%2BFAAJK0GCSH6-U4AZRgWTAtHaEJErjIACMYY4QLpoNQ4YCoN4UoijDgyJIYBjxNjcfiQAiiX5MOlcEFgABtYAleLUbK5EAAMwgECqLBBjRyThRMicLDmAF05nTEftwUSDv8gSCwYqTudLsDQSr6UgACoNWgAMXwMPxhPpxjoEHNEDA1HQ2sRJLJGNK5RwBLtRmRaNcGI2Wx2AJYwRgADoPDHKiwpjGZqF-QH2Mr3QM1XrNVFsx1dQD9VrbXaTWbLdaWJ7yU4AII4CC0CybQEAEWlQfRjebrc2LE70qz5cRuZL%2BcLtWLGoN06Lpsa1dcVzrGNu%2BBYrnwOAA1t3Ub2QA3MFuSLuDyAWKOMywJ3Oy3fZ6WXhA3h9IZgyMpgN8FzOS4Wlaq61qS9YgJK6AwAA%2Bt%2BraHsGGJQTALDaGQCHXreGYPqWhoQi%2BU5jqq0KwvCxH0j2IZOCQMA3K4aHUGI2DQemd5sOuTh2MxECsbSFFErYTKfGyHKEFyPJOAKQogImoq0OKkFStecoKgJvzYQGuFEc%2B6p4QBqRIKRMBwgZtRURi5CFHy1AMZw1j4G6-HsewnEgMi1m2TBUGOWx7FCdsInspybySfygrCvJimSmI0qqWZviaRW%2Bx6TpyXqvhRb7Jw3SZTEBw5T0fQAaE1TqewcxpuVampHlbAAPKYDAfoCQVWg4LoUgyBQ%2BE1SISUHGgNh4EQpCUDQDCySAhQAq26AIKyswPAALPKQA)

Of course it has a solution of using smaller fields or just one field per line.

**Comment by yceballost** [comment]
Hi @jsmolina!

You can solve this issue including `fullWidth` prop to input fields, like [this](https://mistica-web.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8AlGBnADhAduglgG4wAyAhgJ4QCuALgHwA6OABC0gEJ227lV3M2bTACd8AWzIiKXWjxwBeYIKFtO3XAAUxk6StVsp%2BMgFoAFmSwRM1TAsYgoxgDYQA5g-0GtIjOiUAFACULApMrAaqTmSubgHKEZGqtPi0zjDwLA4AcjAA7izEIgS4DgA0XkksMAAetCJkmQGVVewcEDUwUOGtSZwdLJhkUE44bkoAjAAMAL49vX1OhPQAajDF%2BLgsU0gA9EvzCwZIAOJiUCxuZHbAEwBMMyyQztTieEoPh0fHZ-hQAJK0GDiL7fY4AZRg6TAtBaYNUADNqM5nAB1P60Mxw%2BFsZxkABGUPsIG00GoMP%2BUE8iRxBhwZHEMGJojJFKpIGxOOsKVw-mAAG1gIUYtQMlkQAiIBByiwgXVMg58VIHCwZgBdGac767UHwva-AFAkFa3qnc6A4G6nFIAAqtVoADF8FCqTTaUIkSj0VBMSbvmQ6BAHRAwNR0H6jvTGcSiiUcNT3ZE8YTnMS1hstn8WAEYAA6Ny5sosCa5qZBBOJtg6iN9XYGi3G%2BNuvX1o1WvV2upOl013qetEYrHN91RpkOACCOAgmPW-wAIhXK7iCUSJ1OZyIWPPF4nq8PvvrzW3ez8j5aT5Fbfbu84Lv3vZiWKPidd8CxnPgcABrFXJ1cgcdMDfEhPx-EAWD3StDz%2BBt2yOM0YKNJ4IBeN5wSGRRgE%2BC9jk7R1nVvHDInvQciLpBkxxAJV0BgAB9IZfQ5fccT-VNFUsGAWE0MhGLIqs4IWaDDXPZjBNbESl3YSFoVhUSwRIn0h0k5cU2JEgYCuZwuOoERsBondK2fBwbF0iB9KY5SWG5TZ3gFIVCBFMUHElaUQCLOVaAVKjlXA9VNTkhZIMTITYLIhDhMbSSkGkmAYT4j1kQHRT4pU-9yDyT0WA4Sx8HDCzlKMkA8Qy5FaOo3KDMTazeSUQVhReJyJSlGUPK8pURBVPz4qC90QuPALLzrc4BMG9oahGoQ9jGroJqCCpRJmct938yJ2wAeUwGAm1GjQcG0CQpAoL4Vog8I9jQKw8CIUhKBoBhmDckAZ0ZdAEH5EA1ruWjcjyEA1TKEA8kHV74H5aY7gAFg1IA)

Is it enough to close this discussion? If not, can you provide more design examples to understand the need? :)

We would like to avoid opening the width of modals because we want to avoid complex design inside dialogs. If you need more space probably you need to include this information in a single page instead of a modal

wdyt?

**Reply by jsmolina** [reply]
hi! Sure, I knew fullwidth or doubleField was a solution, but I was not so creative showing why they should be wider :). 
I basically wanted to take advantage of the width of the screen, but it's ok if it's not done. 

Thanks.
